src/blocks/content-helper/current-post-details/component.tsx

```diff
@@ -225,9 +236,14 @@ function TopReferrersSection( data: PostPerformanceData ) {
 							return null;
 						}
 
+						let referrerUrl = key;
+						if ( key === 'direct' ) {
+							referrerUrl = __( 'Direct', 'wp-parsely' );
+						}
+
 						return (
 							<tr key={ key }>
-								<th title={ key }>{ key }</th>
+								<th title={ referrerUrl }>{ referrerUrl }</th>
```

Comment:
```suggestion
								<th scope="row">{ referrerUrl }</th>
```

There's no need to repeat the content of the table header cell, as the title attribute.

The `scope` attribute is worth adding to the row `th`, since the table is non-regular - that is, the column heading `th` spans two columns, and the column heading `th` should have a `scope="col"` as well.